src/main/readout.go

```diff
@@ -33,7 +33,7 @@ import (
 	"math"
 	"os"
 	"path/filepath"
-	"github.com/nDenerserve/SmartPi/src/smartpi"
```

Comment:
Typically, I'm working on my own fork.
Including an absolute path to github breaks my system.
How to handle?